cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -141,6 +145,24 @@ func (o *AddOrUpdateOptions) Run(args []string) error {
 		return err
 	}
 
+	if o.CreateNamespace {
+		coreClient, err := o.depsFactory.CoreClient()
+		if err != nil {
+			return err
+		}
+
+		namespace := &v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: o.NamespaceFlags.Name,
+			},
+		}
+
+		_, err = coreClient.CoreV1().Namespaces().Create(context.Background(), namespace, metav1.CreateOptions{})
+		if err != nil {
+			o.ui.PrintLinef("The namespace %s already exists", o.NamespaceFlags.Name)
```

Comment:
We should probably return the error if it's *not* of type [IsAlreadyExists](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#IsAlreadyExists) and ignore if it already exists.

Reply:
+1 to Praveens comment. 
We do something similar over [here](https://github.com/carvel-dev/kapp-controller/blob/develop/cli/pkg/kctrl/cmd/package/installed/create_or_update.go#L585-L595)

However, in the example we attempt an `update` if the service already exists. Where as here, we return the error if the error is `!IsAlreadyExists()`

Reply:
Good point! I changed it as suggested.